wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -52,13 +50,6 @@ public final class ZMUserSession: NSObject {
     private(set) var isNetworkOnline = true
 
     public private(set) var coreDataStack: CoreDataStack!
-    private let apiServiceFactory: APIServiceFactory
-    public var apiService: APIServiceProtocol? {
-        guard let clientId = selfUserClient?.remoteIdentifier else {
-            return nil
-        }
-        return apiServiceFactory(clientId, userId)
-    }
```

Comment:
Here's an example of an additional api service that was created to power some new use cases, we don't want it anymore because the user session has a network objects already (in the "component" objects).